core/output/output.go

```diff
@@ -117,6 +117,20 @@ type Output interface {
 	//  output.EndOperation(true)
 	EndOperation(success bool)
```

Comment:
Shall we add something like this:

```go
// Deprecated: EndOperation exists for historical compatibility
// and should not be used. To conclude the operation, use
// EndOperationWithStatus method.
```

Reply:
Good point. On it.

Reply:
Added in d502a02e303db49c609db0b53fe48b0e4788fc47